build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    version "5.2.0-SNAPSHOT"
+    version "5.2.1-SNAPSHOT"
```

Comment:
Java 11 to 17 is a major change. Bump to `6.0.0-SNAPSHOT`.

Reply:
Done